app/src/main/java/org/mozilla/fenix/StrictModeManager.kt

```diff
@@ -16,16 +16,24 @@ object StrictModeManager {
 
     /***
      * Enables strict mode for debug purposes. meant to be run only in the main process.
-     * @param setPenaltyDialog boolean value to decide setting the dialog box as  a penalty.
+     * @param setPenaltyDeath boolean value to decide setting the penaltyDeath as a penalty.
+     * @param setPenaltyDialog boolean value to decide setting the dialog box as a penalty.
+     * Note: dialog penalty cannot be set with penaltyDeath
      */
-    fun enableStrictMode(setPenaltyDialog: Boolean) {
+    fun enableStrictMode(setPenaltyDeath: Boolean, setPenaltyDialog: Boolean = false) {
```

Comment:
I think just leaving `setPenaltyDeath` would be preferable

Reply:
This just keeps dialogPenalty as an option, so it will be easier to switch between them in the future. Right now this code enforces death penalty.

Reply:
That's true. I guess we can keep it for now and remove it if we see no one uses it 😄  sorry for the late reply!